userspace/libsinsp/plugin.cpp

```diff
@@ -712,6 +713,18 @@ void sinsp_plugin::validate_init_config_json_schema(std::string& config, std::st
 	}
 }
 
+void sinsp_plugin::set_config(const std::string& config)
+{
+	std::string conf = config;
```

Comment:
```suggestion
	if (!m_inited)
	{
		throw sinsp_exception(std::string(s_not_init_err) + ": " + m_name);
	}
	std::string conf = config;
```
I would enforce the plugin to be already initialized as a requirement.